changelog.d/1113.breaking.md

```diff
@@ -0,0 +1,5 @@
+Currently if a grok filter as part of a `parse_groks` operation fails to apply due to an error against the input, the error is not surfaced and the field the filter applies to is not parsed, leading to data loss.
+
+This fix corrects that to surface errors while applying the grok filters.
+
+Existing parse_grok rules may need to be adjusted in order to correct filter queries that are now correctly surfaced as errors.
```

Comment:
Should this be marked as breaking (`!`) in the summary line?

Reply:
👍  yes thanks for flagging. updated.